accounts-db/src/accounts_db.rs

```diff
@@ -4309,6 +4307,34 @@ impl AccountsDb {
             .fetch_add(time.as_us(), Ordering::Relaxed);
     }
 
+    pub(crate) fn unref_shrunk_dead_accounts<'a>(
+        &self,
+        unrefed_pubkeys: impl Iterator<Item = &'a Pubkey>,
```

Comment:
this should be `pubkeys` or `pubkeys_to_unref`

Reply:
fixed.